docs/news/unreleased/other-distro-build.md

```diff
@@ -0,0 +1,145 @@
+# Building images for other distributions
+
+Previously osbuild-composer could only build images for the same distribution
+as the host. With the addition of the distro field in blueprint it is now
+possible to build for any supported distribution shipped with osbuild-composer.
+
+
+## New API route: /distros/list
+
+The API now supports listing the available distributions. It will return a JSON
+object listing the installed distro names that can be used by blueprints,
+sources, and the optional `?distro=` selection on API routes.
+
+eg. `curl --unix-socket /run/weldr/api.socket http://localhost/api/v1/distros/list`
+
+    {
+        "distros": [
+            "centos-8",
+            "fedora-32",
+            "fedora-33",
+            "rhel-8",
```

Comment:
Could we rename this `rhel-83`? The current naming is misleading (my bad), so would be a shame to expose it in an API that didn't yet expose it. We probably need to keep `rhel-8` as an alias in the koji/cloud APIs just in case it is being used still though.

Reply:
If there is a valid distro named rhel-83 with a valid repository file it will, but right now osbuild-composer is aliasing those to rhel-8. The list of distributions is from DistroRegistry, with the ones without actual repo json files filtered out.
I also think that `rhel-8` may be useful as a "latest rhel-8 content" source since it points to the cdn url without a minor version.

In other words :) I think the code is fine, it's the repository data that needs to be updated to include repos for each minor release.

Reply:
I agree, the code is good as-is. However, as this is exposing these names for the first time through the weldr API, I wonder if this PR should also do the renaming so that it will be exposed "correctly"? We could add back a `rhel-8` if we want a generic one, but for now I think we should aim to just expose the distros we have with the most correct names.